src/ingest_validation_tools/directory-schemas/seqfish-v0.yaml

```diff
@@ -1,5 +1,65 @@
 files:
   -
-    pattern: 'TODO'
-    description: 'Directory structure not yet specified.'
-    required: True
\ No newline at end of file
+    pattern: 'codebook/channel_[0-9]+\.(tsv|csv)'
+    description: 'Barcodes.'
+    required: True
+    is_qa_qc: False    
+  -
+    example: 'HybCycle_10/MMStack_Pos12.ome.tif'
+    pattern: 'HybCycle_[0-9]+/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Raw OME-TIFF images with a variable number of field of views, and slices per tissue.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'initial_background/MMStack_Pos11.ome.tif'
+    pattern: 'initial_background/MMStack_Pos[0-9]+\.ome\.tif'
+    description: 'Raw OME-TIFF image of image without probes before imaging hybridizations; used for background and autofluorescence removal.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'final_background/MMStack_Pos11.ome.tif'
+    pattern: 'final_background/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Raw OME-TIFF image of image without probes after all other hybridizations; used for background and autofluorescence removal.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'segmentation/MMStack_Pos3.ome.tif'
+    pattern: 'segmentation/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Labels TIFF image for segmentation.'
+    required: True
+    is_qa_qc: False
+  -
+    example: 'segmentation/raw_images/dapi_Pos19.tif'
+    pattern: 'segmentation/raw_images/dapi_Pos[0-9]+\.(tif|tiff)'
+    description: 'DAPI TIFF image.'
+    required: True
+    is_qa_qc: False
+  -
+    example: 'segmentation/raw_images/membrane_Pos52.tif'
+    pattern: 'segmentation/raw_images/membrane_Pos[0-9]+\.(tif|tiff)'
+    description: 'Membrane TIFF image.'
+    required: False
+    is_qa_qc: False
+  -
+    pattern: 'omexml'
+    description: 'Folder containing metadata for each image.'
+    required: False
+    is_qa_qc: False
+  -
+    example: 'extras/SlideExplorer.tif'
+    pattern: 'extras/SlideExplorer\.tif'
+    description: 'Overview TIFF image for all field of views.'
+    required: False
+    is_qa_qc: False
+  -
+    example: 'positions.pos'
+    pattern: 'positions\.pos'
+    description: 'Micromanager coordinates of each field of view tile.'
```

Comment:
Can you point to anything that describes the expected structure of a `.pos` file? Is there more info in the long assay doc?

Reply:
@mccalluc no, I don't.

Reply:
Micro-Manager (aka µManager) is an established tool and the position list is a standard file that can be read and written using that software. See e.g., https://micro-manager.org/Version_2.0_Users_Guide#stage-position-list or https://valelab4.ucsf.edu/~MM/doc-2.0.0-gamma/mmstudio/org/micromanager/PositionList.html. I don't think that we need to document it separately.